frontend/src/features/projects/api-service.ts

```diff
@@ -0,0 +1,34 @@
+import queryString from 'query-string';
+import CustomAxios from 'customAxios';
+import { IPagedItems } from 'interfaces';
+import { ENVIRONMENT } from 'constants/environment';
+import { IProjectFilter, IProject } from './list/interfaces';
+
+const { apiUrl: basePath } = ENVIRONMENT;
+
+const API_ENDPOINTS = {
+  search: (filter: IProjectFilter) =>
+    `${basePath}/projects/search/page?${filter ? queryString.stringify(filter) : ''}`,
+  delete: (projectNumber: string) => `${basePath}/projects/disposal/${projectNumber}`,
+};
+
+const getProjectList = async (filter: IProjectFilter): Promise<IPagedItems<IProject>> => {
+  const url = API_ENDPOINTS.search(filter);
+  const response = await CustomAxios().get<IPagedItems<IProject>>(url);
```

Comment:
This pattern works but there is no error handling here. That'll end up dumping the user back to the mapview using the existing error boundary in the event of an error, is that intended?